extension/jdtls.ext/com.microsoft.gradle.bs.importer/src/com/microsoft/gradle/bs/importer/GradleBuildServerBuildSupport.java

```diff
@@ -513,7 +513,8 @@ private void setProjectJdk(Map<IPath, IClasspathEntry> classpathMap, List<BuildT
             javaProject.setOption(JavaCore.COMPILER_COMPLIANCE, targetCompatibility);
         }
 
-        if (!StringUtils.isEmpty(jvmBuildTarget.getJavaHome())) {
+        if (!StringUtils.isEmpty(jvmBuildTarget.getJavaHome())
```

Comment:
!StringUtils.isEmpty() -> StringUtils.isNotBlank()

Reply:
Addressed in https://github.com/microsoft/vscode-gradle/pull/1594/commits/4bb27a8263ad828e89eddcb0a74bed6c8a24a3e5